sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -581,33 +581,32 @@ def test_multinomial():
     assert multinomial(1, 1, 1) == 6
     assert multinomial(S.Half, S.Half).equals(4/pi)
 
-    assert unchanged(multinomial, k, n + k)
+    assert multinomial(k, n + k) == binomial(n + 2*k, k)
     assert multinomial(k, n + k).rewrite(binomial
-        ) == binomial(n + 2*k, n + k)
+        ) == binomial(n + 2*k, k)
     # don't rewrite if an arg sign (e.g. sign(n - k)) could be negative
     m = multinomial(k, n - k)
     assert m.rewrite(binomial) == m
 
     assert multinomial(n, n).is_integer
     assert multinomial(nt, k).is_integer is False
 
-    assert expand_func(multinomial(n, -2)) == 0
-    assert (expand_func(multinomial(n, -pi))).equals(
-        (pi - n)*gamma(n - pi)/(pi*n*gamma(-pi)*gamma(n)))
+    assert multinomial(n, -2) == 0
+    assert multinomial(n, -pi).rewrite(gamma
+        ) == gamma(n - pi + 1)/(gamma(1 - pi)*gamma(n + 1))
     assert expand_func(multinomial(n, 2)) == (n + 1)*(n + 2)/2
     assert expand_func(multinomial(n, 2, 3)) == (n + 1)*(
         n + 2)*(n + 3)*(n + 4)*(n + 5)/12
-    assert expand_func(multinomial(n, n + 3)
-        ) == binomial(2*n + 3, n + 3)
+    assert multinomial(n, n + 3) == binomial(2*n + 3, n)
```

Comment:
@sylee957 , what do you think about always making the 2-arg case return as `binomial`; the geometric parallel is returning a Triangle for any 3-non-collinear-points Polygon.